Wikipedia/Code/SectionEditorViewController.swift

```diff
@@ -136,6 +147,31 @@ class SectionEditorViewController: ViewController {
         navigationController?.setNavigationBarHidden(false, animated: false)
     }
     
+    private func removeScriptMessageHandlers() {
+        
+        guard let userContentController = webView?.configuration.userContentController,
+            !scriptMessageHandlers.isEmpty else {
+                return
+        }
+        
+        for handler in scriptMessageHandlers {
+            userContentController.removeScriptMessageHandler(forName: handler.name)
+        }
+        
+        scriptMessageHandlers.removeAll()
+    }
+    
+    private func addScriptMessageHandlers(to contentController: WKUserContentController) {
+        
+        guard !scriptMessageHandlers.isEmpty else {
```

Comment:
Not a bug, but is this necessary?  If the array is empty, we just won't iterate over anything, no?

Reply:
Good point! I removed this.